### DISCUSSION TITLE
Use Isolate in front-end templates

### DISCUSSION THREAD

**Discussion by patbrumfield** [trendyminds/isolate#42] [Q&A] [answered]
Is it possible to access any of Isolate's logic via front-end templates/twig? I'm looking to create a front-end experience where users log in and then have access to only specific entries. Open to any ideas here!

**Comment by aaronbushnell** [comment, marked as answer]
Hey @patbrumfield, you should be able to use any of the variables in [IsolateVariable.php](https://github.com/trendyminds/isolate/blob/master/src/variables/IsolateVariable.php) on the front-end. Not sure if any of them are useful for your particular case, but you can call them like...

```twig
{{ craft.isolate.getUserEntries(currentUser.id) }}
{{ craft.isolate.getIsolateEntries(currentUser.id) }}
```

**Reply by patbrumfield** [reply]
This sounds perfect, thanks for the quick reply @aaronbushnell !

**Reply by aaronbushnell** [reply]
You bet, @patbrumfield! Have a wonderful day 🙌
